diff --git hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-common/src/main/java/org/apache/hadoop/mapred/LocalClientProtocolProvider.java hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-common/src/main/java/org/apache/hadoop/mapred/LocalClientProtocolProvider.java
index cead434..7029593 100644
--- hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-common/src/main/java/org/apache/hadoop/mapred/LocalClientProtocolProvider.java
+++ hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-common/src/main/java/org/apache/hadoop/mapred/LocalClientProtocolProvider.java
@@ -40,8 +40,9 @@ public ClientProtocol create(Configuration conf) throws IOException {
     }
     String tracker = conf.get(JTConfig.JT_IPC_ADDRESS, "local");
     if ("local".equals(tracker)) {
-      conf.setInt("mapreduce.job.maps", 1);
-      return new LocalJobRunner(conf);
+      //conf.setInt("mapreduce.job.maps", 1);
+      //return new LocalJobRunner(conf);
+      return null;
     } else {
 
       throw new IOException("Invalid \"" + JTConfig.JT_IPC_ADDRESS
